### DISCUSSION TITLE
how to set size for tables row

### DISCUSSION THREAD

**Discussion by alexander-naumov** [community/community#122333] [Programming Help]
Hello everyone,

I just want to ask about formatting the tables in Markdown. Since CSS is disabled, I would like to ask how to set the size of rows in tables. For example, I would set 25% for size row. So, the width will always be 25%, no matter what text is in the following rows of the table.

Thank you.

Cheers,
Alex

**Comment by abdullah-dev5** [comment]
you can't directly set the width of rows in a table like you would with CSS. However, you can achieve a similar effect by using HTML within your Markdown document. Here's how you can do it:




<table style="width:100%">
  <tr style="width:25%">
    <td>Row 1</td>
  </tr>
  <tr style="width:25%">
    <td>Row 2</td>
  </tr>
  <tr style="width:25%">
    <td>Row 3</td>
  </tr>
  <tr style="width:25%">
    <td>Row 4</td>
  </tr>
</table>

In this example, each <tr> tag has an inline style specifying its width as 25%. This will ensure that each row occupies 25% of the table's width regardless of the content.

![image](https://github.com/community/community/assets/113471263/383f20b6-35fa-42d0-ac2c-fb00d83bc325)

**Reply by alexander-naumov** [reply]
Thanks for reply.

I don’t know why, but in this case it’s impossible to insert action badge/icons into the table...

**Comment by tstran155** [comment]
In markdown, formatting options for tables are fairly limited because markdown itself does not support CSS or direct styling attributes. Here are two common setups to insert action badge/icons into the table:

- using markdown in markdown tables:
```
| Column 1    | Column 2           |
| ----------- | ------------------ |
| ![](url_to_your_badge) | Data 2 |
| Data 3      | ![](url_to_your_badge) |
```
- using html in markdown:
```
<table>
  <tr>
    <th>Column 1</th>
    <th>Column 2</th>
  </tr>
  <tr>
    <td><img src="url_to_your_badge" alt="Description" /></td>
    <td>Data 2</td>
  </tr>
  <tr>
    <td>Data 3</td>
    <td><img src="url_to_your_badge" alt="Description" /></td>
  </tr>
</table>
```

**Comment by goldieowner17** [comment]
Thanks for posting in the GitHub Community, @alexander-naumov alexander-naumov! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. This question is better suited in "Programming Help"

I've gone ahead and moved this to the correct category for you. Good luck!
